### DISCUSSION TITLE
disallow updates but allow to clone existing item

### DISCUSSION THREAD

**Discussion by wc-matteo** [directus/directus#8816] [Archive]
Is there a way to not allow the user to make changes to existing items but allow them to create a new or clone an existing item?

---

btw what about having a clone button in the collection detail page (next to delete/archive)?

**Comment by joselcvarela** [comment]
Hello @wc-matteo 
Have you checked this: https://docs.directus.io/guides/permissions/#permissions ?
About the clone, next to tick button on top (for saving) you have 3 dots. If you click them, a dropdown will appear and as an option "Saves as new"

**Reply by wc-matteo** [reply]
> About the clone, next to tick button on top (for saving) you have 3 dots. If you click them, a dropdown will appear and as an option "Saves as new"

I mean in the page that lists all of the items, not in the item editor

> Have you checked this: https://docs.directus.io/guides/permissions/#permissions 

Yes. If I remove the update permission, the user has no way to make the "Save as new" button appear

**Comment by rijkvanzanten** [comment]
A duplication in the app would effectively be a read+create operation, so based on the existing permissions that is theoretically possible. That being said, the app doesn't currently have a dedicated button for duplicating an item, so that would be a new feature request 👍🏻

**Reply by wc-matteo** [reply]
https://github.com/directus/directus/discussions/7626
